custom_components/uponor/sensor.py

```diff
@@ -12,6 +12,7 @@ async def async_setup_entry(hass, entry, async_add_entities):
     for thermostat in hass.data[DOMAIN]["thermostats"]:
         if state_proxy.has_floor_temperature(thermostat):
             entities.append(UponorFloorTemperatureSensor(state_proxy, thermostat))
+            entities.append(UponorHumiditySensor(state_proxy, thermostat))
```

Comment:
This should be fixed, this is not correct. Now it adds a humidity sensor to things if it detects a floor temp sensor, it should detect a humidity sensor instead.